test/server/cards/03_TWI/events/CaughtInTheCrossfire.spec.ts

```diff
@@ -0,0 +1,58 @@
+describe('Caught In The Crossfire', function() {
+    integration(function(contextRef) {
+        describe('Caught In The Crossfire\'s event ability', function() {
+            beforeEach(function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['caught-in-the-crossfire'],
+                        groundArena: ['atst', 'battlefield-marine'],
+                    },
+                    player2: {
+                        groundArena: ['clan-wren-rescuer'],
+                        spaceArena: ['cartel-spacer', 'headhunter-squadron', 'avenger#hunting-star-destroyer'],
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true }
+                    }
+                });
+            });
+
+            it('should allow to pick 2 enemy units in the same arena and those units will deal damage to each other', () => {
```

Comment:
nit, cleanliness: could roll the describe and beforeEach all into one big `it()` case. Same for Merciless test